### DISCUSSION TITLE
Support for LIN Bus and Microchip LIN Serial Analyzer Log File?

### DISCUSSION THREAD

**Discussion by DERuth58** [collin80/SavvyCAN#395] [Q&A]
I see on the README page, that SavvyCAN supports Microchip logfiles.  I assume that refers to the Microchip CAN Serial Analyzer logfiles.

Does SavvyCAN support the LIN bus, specifically the Microchip LIN Serial Analyzer, and/or importing and analyzing logfiles from it?  If not currently, is that something that has a potential of being supported in the future?

The reason I ask:
I currently have a Microchip LIN Serial Analyzer, that I've used a bit with the Microchip native SW.  I will be buying a CSS CL2000 CAN logger/interface and using it with the SavvyCAN SW.  It would be nice to only have to learn a single new SW package that works with both LIN and CAN.
Thanks, Doug

**Comment by aaronllowe** [comment]
Bringing this back from the dead because this would be a great addition to SavvyCan's feature set. There are practically no free LINbus debug tools similar to SavvyCAN that make reverse engineering a LINbus system easy.

**Reply by NewGlobalElectronics** [reply]
Here we are:
https://github.com/NewGlobalElectronics/SavvyLIN
We are very happy to contribute to SavvyCAN wonderful tool by creating SavvyLIN variant!

**Comment by NewGlobalElectronics** [comment]
Here we are:
https://github.com/NewGlobalElectronics/SavvyLIN
We are very happy to contribute to SavvyCAN wonderful tool by creating SavvyLIN variant!
